ansible/roles/rhacm_assisted_installer/files/agentserviceconfig.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+ namespace: assisted-installer
+ name: agent
+spec:
+ databaseStorage:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: localstorage-sc
```

Comment:
This storage class always exists in your environment?
Or maybe a customization option will be needed here?

Reply:
In this environment that storage class exists, we could certainly parameterize if we need to.